src/constants/api.ts

```diff
@@ -0,0 +1,13 @@
+import EnvVars from './env';
+
+const ApiConstants = {
+  BEARER: `Bearer ${EnvVars.API_BEARER_TOKEN}`!,
+  BASE_URL: EnvVars.API_BASE_URL!,
+  headers: {
+    accept: 'application/json',
+    'content-type': 'application/json',
+    Authorization: EnvVars.API_BEARER_TOKEN!,
```

Comment:
what is `!`?

Reply:
Using process.env.ANY_Variable. gives `string | undefined` so i added `!`